Increase parameter limit for trait beyond 7 argumentsIntoPyNativeFn

## Summary  
  
The current `IntoPyNativeFn` trait implementation has a hard limit of 7 parameters (excluding `&VirtualMachine`), which prevents defining Python methods with more arguments using the `#[pymethod]` attribute.  
  
## Problem  
  
When trying to define a Python method with more than 7 parameters, the compiler fails with:
error[E0277]: the trait bound is not satisfiedfn(..., ..., i32, f32, i32, i32, i32, i32, i32, i32, i32, ...) -> ... {...::orb}: IntoPyNativeFn<...>

  
## Example  
  
This function fails to compile due to having 11 parameters:  
```rust  
#[pymethod]  
fn orb(  
    query_path: String,  
    source_path: String,  
    nfeatures: i32,  
    scale_factor: f32,  
    nlevels: i32,  
    edge_threshold: i32,  
    first_level: i32,  
    wta_k: i32,  
    score_type: i32,  
    patch_size: i32,  
    fast_threshold: i32,  
    vm: &VirtualMachine,  
) -> PyResult<PyObjectRef> { ... }  
While this function with 7 parameters compiles successfully:

#[pymethod]  
fn sift(  
    query_path: String,  
    source_path: String,  
    nfeatures: i32,  
    n_octave_layers: i32,  
    contrast_threshold: f64,  
    edge_threshold: f64,  
    sigma: f64,  
    vm: &VirtualMachine  
) -> PyResult<PyObjectRef> { ... }




Extend the macro to support more parameter combinations, perhaps up to 15-20 parameters.

--- a/vm/src/function/argument.rs
+++ b/vm/src/function/argument.rs
@@ -55,6 +55,10 @@ into_func_args_from_tuple!((v1, T1), (v2, T2), (v3, T3));
 into_func_args_from_tuple!((v1, T1), (v2, T2), (v3, T3), (v4, T4));
 into_func_args_from_tuple!((v1, T1), (v2, T2), (v3, T3), (v4, T4), (v5, T5));
 into_func_args_from_tuple!((v1, T1), (v2, T2), (v3, T3), (v4, T4), (v5, T5), (v6, T6));
+// We currently allows only 6 unnamed positional arguments.
+// Please use `#[derive(FromArgs)]` and a struct for more complex argument parsing.
+// The number of limitation came from:
+// https://rust-lang.github.io/rust-clippy/master/index.html#too_many_arguments
 
 /// The `FuncArgs` struct is one of the most used structs then creating
 /// a rust function that can be called from python. It holds both positional

